packages/magento-product/components/AddProductsToCart/AddProductsToCartSnackbar.tsx

```diff
@@ -35,8 +36,20 @@ export function AddProductsToCartSnackbar(props: AddProductsToCartSnackbarProps)
 
   const items = filterNonNullableKeys(data?.addProductsToCart?.cart.items)
 
+  const productSkus: (string | undefined | null)[] = []
+
+  submittedVariables?.cartItems.forEach((item) => {
+    productSkus.push(item.sku)
+  })
+
+  const productsAdded = items
+    .filter((item) => productSkus.includes(item.product.sku))
+    .map((product) => product.product.name)
+
   const showErrorSnackbar = userErrors.length > 0
 
+  if (disableSnackbar) return null
```

Comment:
Should we rename this disableSuccess so the user still sees an error message when the product couldn't be added and only prevent the success message?